packages/wallet/src/persistence/types.ts

```diff
@@ -88,9 +88,12 @@ export interface WalletStores extends Destroyable {
   stakePools: KeyValueStore<Cardano.PoolId, Cardano.StakePool>;
   protocolParameters: DocumentStore<ProtocolParametersRequiredByWallet>;
   genesisParameters: DocumentStore<Cardano.CompactGenesis>;
-  stake: DocumentStore<StakeSummary>;
-  lovelaceSupply: DocumentStore<SupplySummary>;
   timeSettings: DocumentStore<TimeSettings[]>;
   assets: DocumentStore<Assets>;
   addresses: DocumentStore<GroupedAddress[]>;
 }
+
+export interface NetworkInfoStatsStores extends Destroyable {
```

Comment:
I am not sure I understand why stake and lovelaceSupply are considered 'Stats' (conclusion drawn from the name).
To me it would sound better if they were 'Summary' items: `NetworkInfoSummaryStores`

Reply:
Nice catch, missed some when renaming. Fixed in 4b6f941bcd1eeecdd11f8e33c597c4fbd71991cd (which also amends commit msg) and e2c908df67ea342dc4495242e92dda1f58550d23